nilearn/datasets/atlas.py

```diff
@@ -50,8 +50,8 @@ def fetch_atlas_difumo(dimension=64, resolution_mm=2, data_dir=None, resume=True
         available are {2, 3}. Default=2mm.
 
     data_dir : string, optional
-        Path where data should be downloaded. By default,
-        files are downloaded in home directory.
+        Path where data should be downloaded. if None, files are downloaded 
+        in (nilearn_data) folder in the home directory.
```

Comment:
Thx ! i'm just wondering why you put the directory btw parentheses. Is this a Python convention ?

Reply:
Hello Bertrand, thank you for the review. I don't know if it's a python convention but I used to sort of underline the name of the download folder.

Reply:
Can you identify a more Pythonic way of indicating it ? (Sorry, I have no clue about that).

Reply:
Sure, I'll look for the conventional way tomorrow 👍🏻

Reply:
maybe pointing to [this paragraph in the user guide](https://nilearn.github.io/manipulating_images/input_output.html#fetching-open-datasets-from-internet) or [this function](https://nilearn.github.io/modules/generated/nilearn.datasets.get_data_dirs.html#nilearn.datasets.get_data_dirs) could help?